Logstash/logstash_filebeats.conf

```diff
@@ -1,73 +1,137 @@
 input {
 	beats {
-    		port => 5044
+    		port => "5044"
   	}
 }
 
 filter {
 	if " Event:" not in [message] {
 	  drop {}
 	}
+
+	mutate {
+        gsub => [
+          "message", "LOCAL", "Resident"
+        ]
+	}
+
+	# 2022-04-26 09:01:22,614 Date format parse
 	grok {
-		match => {"message" => " Unique ID: %{GREEDYDATA:eventType},\n Event: %{GREEDYDATA:eventdata}"}
+		ecs_compatibility => disabled
+		match => {
+			'message' => '\[%{TIMESTAMP_ISO8601:logtime}\].*Unique ID: %{GREEDYDATA:eventType},\n Event: %{GREEDYDATA:eventdata}'
+		}
 	}
+
 	json {
+		ecs_compatibility => disabled
 		source => "eventdata"
 		remove_field => ["eventdata","message"]
 	}
-	#geo ip filter to transform the IP address to location
-	geoip {
-    		source => "[event][payloadData][remoteIp]"
-    	}
 
 	#ruby code to convert the comma seperated roles into an array
 	ruby{
-	     code =>"
+		code =>"
+
+		# method to split the supplied string by comma, trim whitespace and return an array
+		def mapStringToArray(strFieldValue)
 
-		 # method to split the supplied string by comma, trim whitespace and return an array
-		 def mapStringToArray(strFieldValue)
+		#if string is not null, return array
```

Comment:
Lets follow standard when commenting. Start with a capital letter and end with a full stop. Add a space between # and first letter.  `# If string is not null, return array.`

Look for other places as well

Reply:
Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/14